config/schedule.rb

```diff
@@ -0,0 +1,9 @@
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# TODO: Should point this at a cron log
+# TODO: configure add-on for this gem: https://elements.heroku.com/addons/scheduler
+# set :output, "/path/to/my/cron_log.log"
+every 2.hours do
+  rake "alarm_api:update_all_forecasts"
+end
```

Comment:
We can decide on a better interval later, I kept it a little short for testing purposes.  Also, I decided on a rake task in case we want the option of running the task on command, and rake is the idiomatic approach for running batch tasks.